sim/common/sod/crafted/phase_3.go

```diff
@@ -10,12 +10,7 @@ import (
 func init() {
 	core.AddEffectsToTest = false
 
-	///////////////////////////////////////////////////////////////////////////
-	//                                 Cloth
-	///////////////////////////////////////////////////////////////////////////
-
-	// Mantle of Insanity
-	core.NewItemEffect(220749, func(agent core.Agent) {
+	echoesOfInsanityEffect := func(agent core.Agent) {
```

Comment:
🔨 We should probably go with top-level functions here. Apparently the Go compiler struggles a bit with nested functions and top-level are more performant

Reply:
Oh I see thanks, I'm new to go so i'll probably miss a lot of things like that